packages/compass-smoke-tests/package.json

```diff
@@ -24,7 +24,7 @@
     "eslint": "eslint",
     "prettier": "prettier",
     "lint": "npm run eslint . && npm run prettier -- --check .",
-    "depcheck": "compass-scripts check-peer-deps && depcheck",
+    "depcheck": "depcheck",
```

Comment:
😬 If this doesn't handle `optionalDependencies`, we should probably add a ticket tracking a fix for that.

Reply:
I think that's for compass plugins, though. Which is probably why it is a separate script and not part of depcheck.

Reply:
I'll ask @gribnoysup  what he thinks.

Reply:
Checking this rather than npm link, removing it as a peer dep.
https://github.com/mongodb-js/compass/pull/6629/commits/5926b93f63987dc61a7d3aff98a8ca0879e1a7f3